### DISCUSSION TITLE
Cruiser as default app in Android

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#609] [Q&A] [answered]
How can I define Cruiser as standard app for kurv.gr, kurviger.de, osm.org, plus.codes and what3words.com and general for curviger or json format?

**Comment by devemux86** [comment]
@DolomitesTom 

Cruiser should be in the suggested apps when opening several **file** formats:
geojson, gpx, gpz, itn, json, kurviger.

If you mean links, Android now allows apps to register domains for exclusive use.
That's why it is difficult to open Google links on other apps except Google Maps.
e.g. plus code is by Google

**Reply by DolomitesTom** [reply]
> Cruiser` should be in the suggested apps when opening several file formats:
geojson, gpx, gpz, itn, json, kurviger.

yes this is working, but I even don't see or can add cruiser as standard app

**Comment by devemux86** [comment, marked as answer]
Tap any supported file with Google Files, the "Open with" dialog appears with options "Just once" and "Always".

If an app was made the default for opening a format, you can undo this in the Android app settings.

<img src="https://github.com/user-attachments/assets/b386cbd5-a9bd-430c-a4aa-f7d343c2c372" width="50%"/>
